requirements-tests.txt

```diff
@@ -1,12 +1,12 @@
 ply
 py
 pytest
+flake8
+pyflakes<2.1.0,>=2.0.0
```

Comment:
(sorry for the driveby) -- if you remove `pyflakes` from this file, `flake8` will automatically resolve the correct version (it has ~an equivalent requirements in its `setup.py`)

Reply:
Ahh OK thanks!

Reply:
without this, you're hitting a bug in `pip` (which doesn't actually implement dependency resolution)

basically first-pass `pip` greedily sees `pyflakes` and says "latest version of pyflakes please!", then later it sees in the second pass that flake8 wants pyflakes>=2.0,<2.1 and says "I've already seen pyflakes, satisfied!" even when it's a conflict :sob:

Reply:
Yeah.... I figured that was happening. :frowning_face: